packages/caravan-psbt/package.json

```diff
@@ -54,18 +54,15 @@
     "@caravan/eslint-config": "*",
     "@caravan/multisig": "*",
     "@caravan/typescript-config": "*",
-    "@inrupt/jest-jsdom-polyfills": "^3.2.1",
-    "@jest/globals": "^29.7.0",
-    "@types/jest": "^29.5.12",
+    "@vitejs/plugin-react": "^4.3.1",
     "eslint-plugin-prettier": "^5.1.3",
-    "jest": "^29.4.1",
     "jsdom": "24.0.0",
     "jsdom-global": "3.0.2",
     "lodash": "^4.17.21",
     "react-silence": "^1.0.4",
```

Comment:
I think you replaced the uses of this right? so can probably uninstall.

Reply:
Yeah, I haven't uninstalled them completely since those modules are shared in other packages, if uninstalled the test suite there will fail. 

I'll remove those unwanted modules once every package gets migrated to vitest : )

Reply:
Huh. That shouldn’t be the case with the monorepo since they should be isolated. Any package that uses this should have it in its own package.json

Reply:
Yeah, every package has its own package.json but the node_modules and package-lock.json files are shared among them

Reply:
Right so you should be able to remove it from THIS package.json, re-run npm install to update the package lock accordingly and then everything else should still worn

Reply:
I've re-run the `npm i` command and pushed the changes in `package-lock.json` file in my latest commit. I've also checked if the tests are running fine and they are.

Reply:
Right I don’t imagine anything has changed. It looks like react-silence is still in this packages package.json. I think you can remove it and then re-run everything and it should be fine. If it’s not working then that means another package needs to be explicit about the react silence dev dependency.

Reply:
I've removed react-silence from package.json of psbt but after doing so the tests in caravan-bitcoin were failing so I've installed the same in its own package.json file